boilerplate/.build/webpack/base.js

```diff
@@ -21,7 +21,7 @@ const path = require('path'),
 const base = {
   target: 'web',
   entry: {
-    app: './src/index.tsx',
+    app: process.env.ie ? ['core-js', './src/index.tsx'] : './src/index.tsx',
```

Comment:
I do not really like that we have references to IE in every file now.
This is not the way i would like it to be.

Still not sure if we should reinclude IE11 support because we had dropped it initially.
At least it would require to have 1 separate webpack file which does all the IE11 stuff and other files should not be touched at all.

Reply:
I could do that too.
Should i merge everything into 1 file or maybe introduce 2. for seperate dev and prod environments maybe?

Reply:
Sorry for the radio-silence, we currently are swamped with work. I will have a look next week and come back to you afterwards.